### PR TITLE
Feat/oss v2 2 metrics

### PR DESCRIPTION
**Pull request by jstirnaman**
Describes new storage metrics in OSS v2.2. Link to metrics from release notes.

- [ ] Signed the [InfluxData CLA](https://www.influxdata.com/legal/cla/)
  ([if necessary](https://github.com/influxdata/docs-v2/blob/master/CONTRIBUTING.md#sign-the-influxdata-cla))
- [x] Rebased/mergeable

### CODE REVIEW

File: content/influxdb/v2.2/reference/internals/metrics.md [hunk 1]
```diff
@@ -1060,9 +1061,462 @@ service_user_new_duration_bucket{method="find_permission_for_user",le="0.005"} 4
 --
 ```
 
+## InfluxDB storage statistics
+
+To learn how InfluxDB writes, stores, and caches data, see [InfluxDB storage engine](/influxdb/v2.2/reference/internals/storage-engine/).
+
+### Bucket measurement number
+
+Gauge of measurement cardinality per bucket.
```
Comment (sanderson) [inline content/influxdb/v2.2/reference/internals/metrics.md:1070]: Number of measurements in a bucket.

File: content/influxdb/v2.2/reference/internals/metrics.md [hunk 2]
```diff
@@ -1060,9 +1061,462 @@ service_user_new_duration_bucket{method="find_permission_for_user",le="0.005"} 4
...
+```sh
+# HELP storage_bucket_series_num Gauge of series cardinality per bucket
+# TYPE storage_bucket_series_num gauge
+storage_bucket_series_num{bucket="0c3dd7d2d97f4b23"} 38
+--
+```
+
+### Cache disk bytes
+
+Gauge of the size of the most recent [snapshot](/influxdb/v2.2/reference/internals/storage-engine/#cache).
```
Comment (sanderson) [inline content/influxdb/v2.2/reference/internals/metrics.md:1096]:
```suggestion
Size (in bytes) of the most recent [snapshot](/influxdb/v2.2/reference/internals/storage-engine/#cache).
```

File: content/influxdb/v2.2/reference/internals/metrics.md [hunk 3]
```diff
@@ -1060,9 +1061,462 @@ service_user_new_duration_bucket{method="find_permission_for_user",le="0.005"} 4
...
+```sh
+# HELP storage_bucket_measurement_num Gauge of measurement cardinality per bucket
+# TYPE storage_bucket_measurement_num gauge
+storage_bucket_measurement_num{bucket="0c3dd7d2d97f4b23"} 4
+--
+```
+
+### Bucket series number
+
+Gauge of series cardinality per bucket.
```
Comment (sanderson) [inline content/influxdb/v2.2/reference/internals/metrics.md:1083]:
```suggestion
Series cardinality per bucket.
```

File: content/influxdb/v2.2/reference/internals/metrics.md [hunk 4]
```diff
@@ -1060,9 +1061,462 @@ service_user_new_duration_bucket{method="find_permission_for_user",le="0.005"} 4
...
+```sh
+# HELP storage_cache_disk_bytes Gauge of size of most recent snapshot
+# TYPE storage_cache_disk_bytes gauge
+storage_cache_disk_bytes{bucket="0c3dd7d2d97f4b23",engine="tsm1",id="561",path="/Users/me/.influxdbv2/engine/data/0c3dd7d2d97f4b23/autogen/561",walPath="/Users/me/.influxdbv2/engine/wal/0c3dd7d2d97f4b23/autogen/561"} 0
+--
+```
+
+### Cache in use bytes
+
+Gauge of current memory consumption of the [cache](/influxdb/v2.2/reference/internals/storage-engine/#cache).
```
Comment (sanderson) [inline content/influxdb/v2.2/reference/internals/metrics.md:1109]:
```suggestion
Current memory consumption (in bytes) of the [cache](/influxdb/v2.2/reference/internals/storage-engine/#cache).
```

File: content/influxdb/v2.2/reference/internals/metrics.md [hunk 5]
```diff
@@ -1060,9 +1061,462 @@ service_user_new_duration_bucket{method="find_permission_for_user",le="0.005"} 4
...
+```sh
+# HELP storage_cache_latest_snapshot Unix time of most recent snapshot
+# TYPE storage_cache_latest_snapshot gauge
+storage_cache_latest_snapshot{bucket="0c3dd7d2d97f4b23",engine="tsm1",id="561",path="/Users/me/.influxdbv2/engine/data/0c3dd7d2d97f4b23/autogen/561",walPath="/Users/me/.influxdbv2/engine/wal/0c3dd7d2d97f4b23/autogen/561"} 1.644269658196893e+09
+--
+```
+
+### Cache writes with dropped points
+
+Counter of [cached](/influxdb/v2.2/reference/internals/storage-engine/#cache) writes that had [rejected points](/influxdb/v2.2/reference/glossary/#rejected-point). Writes with rejected points also increment the [write errors counter (`storage_cache_writes_err`)](#cache-writes-failed).
```
Comment (sanderson) [inline content/influxdb/v2.2/reference/internals/metrics.md:1135]:
```suggestion
Cumulative number of [cached](/influxdb/v2.2/reference/internals/storage-engine/#cache) writes that had [rejected points](/influxdb/v2.2/reference/glossary/#rejected-point). Writes with rejected points also increment the [write errors counter (`storage_cache_writes_err`)](#cache-writes-failed).
```

File: content/influxdb/v2.2/reference/internals/metrics.md [hunk 6]
```diff
@@ -1060,9 +1061,462 @@ service_user_new_duration_bucket{method="find_permission_for_user",le="0.005"} 4
...
+```sh
+# HELP storage_cache_writes_dropped Counter of writes to cache with some dropped points
+# TYPE storage_cache_writes_dropped counter
+storage_cache_writes_dropped{bucket="0c3dd7d2d97f4b23",engine="tsm1",id="561",path="/Users/me/.influxdbv2/engine/data/0c3dd7d2d97f4b23/autogen/561",walPath="/Users/me/.influxdbv2/engine/wal/0c3dd7d2d97f4b23/autogen/561"} 0
+--
+```
+
+### Cache writes failed
+
+Counter of [cached](/influxdb/v2.2/reference/internals/storage-engine/#cache) writes that [failed](/influxdb/v2.0/write-data/troubleshoot/#troubleshoot-failures), inclusive of [cache writes with dropped points (`storage_cache_writes_dropped`)](#cache-writes-with-dropped-points).
```
Comment (sanderson) [inline content/influxdb/v2.2/reference/internals/metrics.md:1148]:
```suggestion
Cumulative number of [cached](/influxdb/v2.2/reference/internals/storage-engine/#cache) writes that [failed](/influxdb/v2.0/write-data/troubleshoot/#troubleshoot-failures), inclusive of [cache writes with dropped points (`storage_cache_writes_dropped`)](#cache-writes-with-dropped-points).
```

File: content/influxdb/v2.2/reference/internals/metrics.md [hunk 7]
```diff
@@ -1060,9 +1061,462 @@ service_user_new_duration_bucket{method="find_permission_for_user",le="0.005"} 4
...
+```sh
+# HELP storage_cache_writes_err Counter of failed writes to cache
+# TYPE storage_cache_writes_err counter
+storage_cache_writes_err{bucket="0c3dd7d2d97f4b23",engine="tsm1",id="561",path="/Users/me/.influxdbv2/engine/data/0c3dd7d2d97f4b23/autogen/561",walPath="/Users/me/.influxdbv2/engine/wal/0c3dd7d2d97f4b23/autogen/561"} 0
+--
+```
+
+### Cache writes total
+
+Counter of total writes to [cache](/influxdb/v2.2/reference/internals/storage-engine/#cache).
```
Comment (sanderson) [inline content/influxdb/v2.2/reference/internals/metrics.md:1161]:
```suggestion
Cumulative number of total writes to [cache](/influxdb/v2.2/reference/internals/storage-engine/#cache).
```

File: content/influxdb/v2.2/reference/internals/metrics.md [hunk 8]
```diff
@@ -1060,9 +1061,462 @@ service_user_new_duration_bucket{method="find_permission_for_user",le="0.005"} 4
...
+```sh
+# HELP storage_cache_writes_total Counter of all writes to cache
+# TYPE storage_cache_writes_total counter
+storage_cache_writes_total{bucket="0c3dd7d2d97f4b23",engine="tsm1",id="561",path="/Users/me/.influxdbv2/engine/data/0c3dd7d2d97f4b23/autogen/561",walPath="/Users/me/.influxdbv2/engine/wal/0c3dd7d2d97f4b23/autogen/561"} 0
+--
+```
+
+### Compactions active
+
+Gauge of currently running [TSM](/influxdb/v2.2/reference/internals/storage-engine/#time-structured-merge-tree-tsm) compactions (by level).
```
Comment (sanderson) [inline content/influxdb/v2.2/reference/internals/metrics.md:1174]:
```suggestion
Number of currently running [TSM](/influxdb/v2.2/reference/internals/storage-engine/#time-structured-merge-tree-tsm) compactions (by level).
```

File: content/influxdb/v2.2/reference/internals/metrics.md [hunk 9]
```diff
@@ -1060,9 +1061,462 @@ service_user_new_duration_bucket{method="find_permission_for_user",le="0.005"} 4
...
+storage_compactions_duration_seconds_bucket{bucket="0c3dd7d2d97f4b23",engine="tsm1",id="567",level="cache",path="/Users/me/.influxdbv2/engine/data/0c3dd7d2d97f4b23/autogen/567",walPath="/Users/me/.influxdbv2/engine/wal/0c3dd7d2d97f4b23/autogen/567",le="+Inf"} 1
+storage_compactions_duration_seconds_sum{bucket="0c3dd7d2d97f4b23",engine="tsm1",id="567",level="cache",path="/Users/me/.influxdbv2/engine/data/0c3dd7d2d97f4b23/autogen/567",walPath="/Users/me/.influxdbv2/engine/wal/0c3dd7d2d97f4b23/autogen/567"} 0.167250668
+storage_compactions_duration_seconds_count{bucket="0c3dd7d2d97f4b23",engine="tsm1",id="567",level="cache",path="/Users/me/.influxdbv2/engine/data/0c3dd7d2d97f4b23/autogen/567",walPath="/Users/me/.influxdbv2/engine/wal/0c3dd7d2d97f4b23/autogen/567"} 1
+--
+```
+
+### Compactions failed
+
+Counter of failed [TSM](/influxdb/v2.2/reference/internals/storage-engine/#time-structured-merge-tree-tsm) compactions (by level).
```
Comment (sanderson) [inline content/influxdb/v2.2/reference/internals/metrics.md:1205]:
```suggestion
Cumulative number of failed [TSM](/influxdb/v2.2/reference/internals/storage-engine/#time-structured-merge-tree-tsm) compactions (by level).
```

File: content/influxdb/v2.2/reference/internals/metrics.md [hunk 10]
```diff
@@ -1060,9 +1061,462 @@ service_user_new_duration_bucket{method="find_permission_for_user",le="0.005"} 4
...
+```sh
+# HELP storage_compactions_failed Counter of TSM compactions (by level) that have failed due to error
+# TYPE storage_compactions_failed counter
+storage_compactions_failed{bucket="ec3f82d1de90eddf",engine="tsm1",id="565",level="1",path="/Users/me/.influxdbv2/engine/data/ec3f82d1de90eddf/autogen/565",walPath="/Users/me/.influxdbv2/engine/wal/ec3f82d1de90eddf/autogen/565"} 0
+--
+```
+
+### Compactions queued
+
+Counter of queued [TSM](/influxdb/v2.2/reference/internals/storage-engine/#time-structured-merge-tree-tsm) compactions (by level).
```
Comment (sanderson) [inline content/influxdb/v2.2/reference/internals/metrics.md:1218]:
```suggestion
Number of queued [TSM](/influxdb/v2.2/reference/internals/storage-engine/#time-structured-merge-tree-tsm) compactions (by level).
```

File: content/influxdb/v2.2/reference/internals/metrics.md [hunk 11]
```diff
@@ -1060,9 +1061,462 @@ service_user_new_duration_bucket{method="find_permission_for_user",le="0.005"} 4
...
+storage_retention_check_duration_bucket{le="10"} 1
+storage_retention_check_duration_bucket{le="+Inf"} 1
+storage_retention_check_duration_sum 0.000351857
+storage_retention_check_duration_count 1
+--
+```
+
+### Shard disk size
+
+Gauge of disk size (in bytes) for the [shard](/influxdb/v2.2/reference/internals/shards/).
```
Comment (sanderson) [inline content/influxdb/v2.2/reference/internals/metrics.md:1257]:
```suggestion
Disk size (in bytes) for a [shard](/influxdb/v2.2/reference/internals/shards/).
```

File: content/influxdb/v2.2/reference/internals/metrics.md [hunk 12]
```diff
@@ -1060,9 +1061,462 @@ service_user_new_duration_bucket{method="find_permission_for_user",le="0.005"} 4
...
+```sh
+# HELP storage_shard_fields_created Counter of the number of fields created
+# TYPE storage_shard_fields_created counter
+storage_shard_fields_created{bucket="0c3dd7d2d97f4b23",engine="tsm1",id="561",path="/Users/me/.influxdbv2/engine/data/0c3dd7d2d97f4b23/autogen/561",walPath="/Users/me/.influxdbv2/engine/wal/0c3dd7d2d97f4b23/autogen/561"} 0
+--
+```
+
+### Shard series
+
+Gauge of the number of series in the [shard](/influxdb/v2.2/reference/internals/shards/) index.
```
Comment (sanderson) [inline content/influxdb/v2.2/reference/internals/metrics.md:1283]:
```suggestion
Number of series in a [shard](/influxdb/v2.2/reference/internals/shards/) index.
```

File: content/influxdb/v2.2/reference/internals/metrics.md [hunk 13]
```diff
@@ -1060,9 +1061,462 @@ service_user_new_duration_bucket{method="find_permission_for_user",le="0.005"} 4
...
+```sh
+# HELP storage_shard_write_count Count of the number of write requests
+# TYPE storage_shard_write_count counter
+storage_shard_write_count{bucket="0c3dd7d2d97f4b23",engine="tsm1",id="561",path="/Users/me/.influxdbv2/engine/data/0c3dd7d2d97f4b23/autogen/561",walPath="/Users/me/.influxdbv2/engine/wal/0c3dd7d2d97f4b23/autogen/561"} 0
+--
+```
+
+### Shard dropped points
+
+Number of [rejected points](/influxdb/v2.2/reference/glossary/#rejected-point) in [shard writes](/influxdb/v2.2/reference/internals/shards/#shard-writes).
```
Comment (sanderson) [inline content/influxdb/v2.2/reference/internals/metrics.md:1309]:
```suggestion
Cumulative number of [rejected points](/influxdb/v2.2/reference/glossary/#rejected-point) in [shard writes](/influxdb/v2.2/reference/internals/shards/#shard-writes).
```

File: content/influxdb/v2.2/reference/internals/metrics.md [hunk 14]
```diff
@@ -1060,9 +1061,462 @@ service_user_new_duration_bucket{method="find_permission_for_user",le="0.005"} 4
...
+```sh
+# HELP storage_shard_series Gauge of the number of series in the shard index
+# TYPE storage_shard_series gauge
+storage_shard_series{bucket="0c3dd7d2d97f4b23",engine="tsm1",id="561",path="/Users/me/.influxdbv2/engine/data/0c3dd7d2d97f4b23/autogen/561",walPath="/Users/me/.influxdbv2/engine/wal/0c3dd7d2d97f4b23/autogen/561"} 38
+--
+```
+
+### Shard writes
+
+Number of [shard write](/influxdb/v2.2/reference/internals/shards/#shard-writes) requests.
```
Comment (sanderson) [inline content/influxdb/v2.2/reference/internals/metrics.md:1296]:
```suggestion
Cumulative number of [shard write](/influxdb/v2.2/reference/internals/shards/#shard-writes) requests.
```

File: content/influxdb/v2.2/reference/internals/metrics.md [hunk 15]
```diff
@@ -1060,9 +1061,462 @@ service_user_new_duration_bucket{method="find_permission_for_user",le="0.005"} 4
...
+```sh
+# HELP storage_shard_write_dropped_sum Counter of the number of points dropped
+# TYPE storage_shard_write_dropped_sum counter
+storage_shard_write_dropped_sum{bucket="0c3dd7d2d97f4b23",engine="tsm1",id="561",path="/Users/me/.influxdbv2/engine/data/0c3dd7d2d97f4b23/autogen/561",walPath="/Users/me/.influxdbv2/engine/wal/0c3dd7d2d97f4b23/autogen/561"} 0
+--
+```
+
+### Shard writes with errors
+
+Number of [shard write](/influxdb/v2.2/reference/internals/shards/#shard-writes) requests with errors.
```
Comment (sanderson) [inline content/influxdb/v2.2/reference/internals/metrics.md:1322]:
```suggestion
Cumulative number of [shard write](/influxdb/v2.2/reference/internals/shards/#shard-writes) requests with errors.
```

File: content/influxdb/v2.2/reference/internals/metrics.md [hunk 16]
```diff
@@ -1060,9 +1061,462 @@ service_user_new_duration_bucket{method="find_permission_for_user",le="0.005"} 4
...
+```sh
+# HELP storage_shard_write_err_count Count of the number of write requests with errors
+# TYPE storage_shard_write_err_count counter
+storage_shard_write_err_count{bucket="0c3dd7d2d97f4b23",engine="tsm1",id="561",path="/Users/me/.influxdbv2/engine/data/0c3dd7d2d97f4b23/autogen/561",walPath="/Users/me/.influxdbv2/engine/wal/0c3dd7d2d97f4b23/autogen/561"} 0
+--
+```
+
+### Points in shard writes with errors
+
+Number of points in [shard write](/influxdb/v2.2/reference/internals/shards/#shard-writes) requests with errors.
```
Comment (sanderson) [inline content/influxdb/v2.2/reference/internals/metrics.md:1335]:
```suggestion
Cumulative number of points in [shard write](/influxdb/v2.2/reference/internals/shards/#shard-writes) requests with errors.
```

File: content/influxdb/v2.2/reference/internals/metrics.md [hunk 17]
```diff
@@ -1060,9 +1061,462 @@ service_user_new_duration_bucket{method="find_permission_for_user",le="0.005"} 4
...
+```sh
+# HELP storage_shard_write_err_sum Counter of the number of points for write requests with errors
+# TYPE storage_shard_write_err_sum counter
+storage_shard_write_err_sum{bucket="0c3dd7d2d97f4b23",engine="tsm1",id="561",path="/Users/me/.influxdbv2/engine/data/0c3dd7d2d97f4b23/autogen/561",walPath="/Users/me/.influxdbv2/engine/wal/0c3dd7d2d97f4b23/autogen/561"} 0
+--
+```
+
+### Points in shard writes
+
+Number of points in [shard write](/influxdb/v2.2/reference/internals/shards/#shard-writes) requests.
```
Comment (sanderson) [inline content/influxdb/v2.2/reference/internals/metrics.md:1348]:
```suggestion
Cumulative number of points in [shard write](/influxdb/v2.2/reference/internals/shards/#shard-writes) requests.
```

File: content/influxdb/v2.2/reference/internals/metrics.md [hunk 18]
```diff
@@ -1060,9 +1061,462 @@ service_user_new_duration_bucket{method="find_permission_for_user",le="0.005"} 4
...
+```sh
+# HELP storage_shard_write_sum Counter of the number of points for write requests
+# TYPE storage_shard_write_sum counter
+storage_shard_write_sum{bucket="0c3dd7d2d97f4b23",engine="tsm1",id="561",path="/Users/me/.influxdbv2/engine/data/0c3dd7d2d97f4b23/autogen/561",walPath="/Users/me/.influxdbv2/engine/wal/0c3dd7d2d97f4b23/autogen/561"} 0
+--
+```
+
+### Shard data size
+
+Gauge of the data size (in bytes) for each [shard](/influxdb/v2.2/reference/internals/shards/).
```
Comment (sanderson) [inline content/influxdb/v2.2/reference/internals/metrics.md:1361]:
```suggestion
Data size (in bytes) for a [shard](/influxdb/v2.2/reference/internals/shards/).
```

File: content/influxdb/v2.2/reference/internals/metrics.md [hunk 19]
```diff
@@ -1060,9 +1061,462 @@ service_user_new_duration_bucket{method="find_permission_for_user",le="0.005"} 4
...
+```sh
+# HELP storage_tsm_files_disk_bytes Gauge of data size in bytes for each shard
+# TYPE storage_tsm_files_disk_bytes gauge
+storage_tsm_files_disk_bytes{bucket="0c3dd7d2d97f4b23",engine="tsm1",id="561",path="/Users/me/.influxdbv2/engine/data/0c3dd7d2d97f4b23/autogen/561",walPath="/Users/me/.influxdbv2/engine/wal/0c3dd7d2d97f4b23/autogen/561"} 4.188743e+06
+--
+```
+
+### Shard files
+
+Gauge of the number of files per [shard](/influxdb/v2.2/reference/internals/shards/).
```
Comment (sanderson) [inline content/influxdb/v2.2/reference/internals/metrics.md:1374]:
```suggestion
Number of files in a [shard](/influxdb/v2.2/reference/internals/shards/).
```

File: content/influxdb/v2.2/reference/internals/metrics.md [hunk 20]
```diff
@@ -1060,9 +1061,462 @@ service_user_new_duration_bucket{method="find_permission_for_user",le="0.005"} 4
...
+```sh
+# HELP storage_tsm_files_total Gauge of number of files per shard
+# TYPE storage_tsm_files_total gauge
+storage_tsm_files_total{bucket="0c3dd7d2d97f4b23",engine="tsm1",id="561",path="/Users/me/.influxdbv2/engine/data/0c3dd7d2d97f4b23/autogen/561",walPath="/Users/me/.influxdbv2/engine/wal/0c3dd7d2d97f4b23/autogen/561"} 1
+--
+```
+
+### WAL size
+
+Gauge of the [WAL](/influxdb/v2.2/reference/internals/storage-engine/#write-ahead-log-wal) size (in bytes).
```
Comment (sanderson) [inline content/influxdb/v2.2/reference/internals/metrics.md:1387]:
```suggestion
[WAL](/influxdb/v2.2/reference/internals/storage-engine/#write-ahead-log-wal) size (in bytes).
```

File: content/influxdb/v2.2/reference/internals/metrics.md [hunk 21]
```diff
@@ -1060,9 +1061,462 @@ service_user_new_duration_bucket{method="find_permission_for_user",le="0.005"} 4
...
+```sh
+# HELP storage_wal_size Gauge of size of WAL in bytes
+# TYPE storage_wal_size gauge
+storage_wal_size{bucket="0c3dd7d2d97f4b23",engine="tsm1",id="561",path="/Users/me/.influxdbv2/engine/data/0c3dd7d2d97f4b23/autogen/561",walPath="/Users/me/.influxdbv2/engine/wal/0c3dd7d2d97f4b23/autogen/561"} 0
+--
+```
+
+## WAL write attempts
+
+Number of write attempts to the [WAL](/influxdb/v2.2/reference/internals/storage-engine/#write-ahead-log-wal).
```
Comment (sanderson) [inline content/influxdb/v2.2/reference/internals/metrics.md:1400]:
```suggestion
Cumulative number of write attempts to the [WAL](/influxdb/v2.2/reference/internals/storage-engine/#write-ahead-log-wal).
```

File: content/influxdb/v2.2/reference/internals/metrics.md [hunk 22]
```diff
@@ -1060,9 +1061,462 @@ service_user_new_duration_bucket{method="find_permission_for_user",le="0.005"} 4
...
+```sh
+# HELP storage_wal_writes Number of write attempts to the WAL
+# TYPE storage_wal_writes counter
+storage_wal_writes{bucket="0c3dd7d2d97f4b23",engine="tsm1",id="561",path="/Users/me/.influxdbv2/engine/data/0c3dd7d2d97f4b23/autogen/561",walPath="/Users/me/.influxdbv2/engine/wal/0c3dd7d2d97f4b23/autogen/561"} 0
+--
+```
+
+## WAL failed write attempts
+
+Number of failed write attempts to the [WAL](/influxdb/v2.2/reference/internals/storage-engine/#write-ahead-log-wal).
```
Comment (sanderson) [inline content/influxdb/v2.2/reference/internals/metrics.md:1413]:
```suggestion
Cumulative number of failed write attempts to the [WAL](/influxdb/v2.2/reference/internals/storage-engine/#write-ahead-log-wal).
```

File: content/influxdb/v2.2/reference/internals/metrics.md [hunk 23]
```diff
@@ -1060,9 +1061,462 @@ service_user_new_duration_bucket{method="find_permission_for_user",le="0.005"} 4
...
+storage_writer_req_points_bucket{path="/Users/me/.influxdbv2/engine",le="100000"} 18
+storage_writer_req_points_bucket{path="/Users/me/.influxdbv2/engine",le="+Inf"} 18
+storage_writer_req_points_sum{path="/Users/me/.influxdbv2/engine"} 125787
+storage_writer_req_points_count{path="/Users/me/.influxdbv2/engine"} 18
+--
+```
+
+## Shard write request timeouts
+
+Number of [shard write](/influxdb/v2.2/reference/internals/shards/#shard-writes) request timeouts.
```
Comment (sanderson) [inline content/influxdb/v2.2/reference/internals/metrics.md:1506]:
```suggestion
Cumulative number of [shard write](/influxdb/v2.2/reference/internals/shards/#shard-writes) request timeouts.
```

**Review by sanderson — CHANGES_REQUESTED** [review]
I think your branch has some changes that haven't been merged into the `oss-2.2.base` branch. There are files being deleted that definitely should not be deleted.

**Comment by jstirnaman** [comment]
> I think your branch has some changes that haven't been merged into the `oss-2.2.base` branch. There are files being deleted that definitely should not be deleted.

@sanderson Yep, sorry about that.

**Pull request merged**